app/consumers/compliance_reports_consumer.rb

```diff
@@ -6,40 +6,38 @@ class ComplianceReportsConsumer < ApplicationConsumer
   subscribes_to Settings.platform_kafka_topic
 
   def process(message)
-    @value = JSON.parse(message.value)
+    @msg_value = JSON.parse(message.value)
```

Comment:
Any reason for this variable rename?

Reply:
`value` has got to be one of the worst variable names there is (after `foo`, `bar`, etc.), especially as an instance variable. I think this adds to the readability of the code by adding a bit more context. I'm certainly open to alternate names, but not `value`. :)